rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -218,11 +226,7 @@ impl Parse for CountersRawRow {
         }
         let retention_days = enforce_retention(Some(from.retention_days), &config.env_config);
 
-        let record_meta = match from.use_case_id.as_str() {
-            "escalating_issues" => Some(0),
-            "metric_stats" => Some(0),
-            _ => Some(1),
-        };
+        let record_meta = should_record_meta(from.use_case_id.clone());
```

Comment:
And then this can change to pass by reference. That way there isn't necessarily a need to clone() this
```suggestion
        let record_meta = should_record_meta(from.use_case_id.as_str());
```